python/modem-firmware-1.3+/command_interface.py

```diff
@@ -82,6 +82,10 @@ def __init__(self, message, exit_code):
         self.exit_code = exit_code
 
 class ATCommandInterface(CredentialCommandInterface):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.shell = False
```

Comment:
No need to overload the constructor, you can just write

`shell = False` directly in the class

It's fine, since it's an immutable type it won't propagate

I tend to prefer this wherever possible since constructor overloading can quickly become a nightmare